### DISCUSSION TITLE
Typo in wiki page, Undefined method 'get'.

### DISCUSSION THREAD

**Discussion by yudistirasd** [ivanwilliammd/satusehat-integration#117] [Ideas]
![image](https://github.com/ivanwilliammd/satusehat-integration/assets/25135334/f6d1abb0-d3d4-4dfc-80a6-114bbd660c4f)

https://github.com/ivanwilliammd/satusehat-integration/wiki/Usage#contoh-get-by-id

**Comment by SyaefulKai** [comment]
Pakai $client->get_by_id() pak

**Comment by yudistirasd** [comment]
iya mas udah ketemu, tadi sempet bingung aja. wiki sama codenya beda hehehe.

**Comment by ivanwilliammd** [comment]
Done, revised at https://github.com/ivanwilliammd/satusehat-integration/wiki/Usage

Thanks
